etc/kayobe/overcloud-dib.yml

```diff
@@ -71,7 +71,7 @@ overcloud_dib_host_packages_extra:
 overcloud_dib_git_elements_extra:
   - repo: "https://github.com/stackhpc/stackhpc-image-elements"
     local: "{{ source_checkout_path }}/stackhpc-image-elements"
-    version: "v1.6.0"
+    version: "v1.6.1"
```

Comment:
Should this be in a separate change?

Reply:
Builds flat out fail without it, so I'd argue it should remain part of this change

Reply:
Ack